docs/tutorials/wiki2/installation.rst

```diff
@@ -273,7 +273,7 @@ On Windows
 
 There should be no output to your console.
 You should now have a ``tutorial.sqlite`` file in your current working directory.
-This is an SQLite database with three tables defined in it, ``alembic_version``, ``models``, and ``master``, where the first two tables each have single record inside of them.
+This is an SQLite database with three tables defined in it, ``alembic_version``, ``models``, and ``main``, where the first two tables each have single record inside of them.
```

Comment:
I ran the cookiecutter and do not see a master/main table created, not sure where that language came from in the tutorial.

Reply:
Yeah, I don't know where that came from, but it was at least 4 years ago. This should read:

```
This is an SQLite database with two tables defined in it, ``alembic_version`` and ``models``, where each table has a single record.

Reply:
```suggestion
This is an SQLite database with three tables defined in it, ``alembic_version``, ``models``, where the first two tables each have single record inside of them.
```

Reply:
That should be this, as I mentioned earlier:

```
This is an SQLite database with two tables defined in it, ``alembic_version`` and ``models``, where each table has a single record.
```

Reply:
Oops thanks for following up on it.